geode-core/src/main/java/org/apache/geode/distributed/LocatorLauncher.java

```diff
@@ -1075,8 +1082,7 @@ private LocatorState stopWithWorkingDirectory() {
     try {
       final ProcessController controller =
           new ProcessControllerFactory().createProcessController(this.controllerParameters,
-              new File(getWorkingDirectory()), ProcessType.LOCATOR.getPidFileName(),
-              READ_PID_FILE_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
+              new File(getWorkingDirectory()), ProcessType.LOCATOR.getPidFileName());
       parsedPid = controller.getProcessId();
 
       // NOTE in-process request will go infinite loop unless we do the following
```

Comment:
There are several opportunities for collapsing multiple catch blocks in this class, such as the try/catch that starts at line 1082